extra/service_startup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+if [[ -e /root/tmp/certbot.sh ]]; then
+    /bin/bash /root/tmp/certbot.sh
```

Comment:
Nevermind my comment, I didn't realize the file will still be there inside the container. The only way to handle this without taking all the certificates will be to handle that outside of the container and just copy the files...